lib/methods/veres-one/veres-one.config.js

```diff
@@ -1,13 +0,0 @@
-module.exports = {
-  hostnames: {
-    'testnet': [
-      'alturas',
-      'frankfurt',
-      'genesis',
-      'mumbai',
-      'saopaulo',
-      'singapore',
-      'tokyo'
-    ].map(nick => `${nick}.testnet.veres.one`)
-  }
-};
```

Comment:
Was this file intentionally removed?

Reply:
Thanks for double-checking. Yes, removed intentionally - it's not really needed in this lib.